### ISSUE TITLE
The shutdown menu bitmap

### ISSUE THREAD

**Issue by kamyk2000** [Open-Shell/Open-Shell-Menu#545]
Would it be possible to have an option to use a separate bitmap for the shutdown popup menu, instead of the main submenu one?  One that doesn't stretch? It would be nice to be able to add icons next to each entry, like an image of zzz for sleep, or a little generic icon of two people for switch user, etc.  As it stands now, from the level of understanding I've gotten to, it will always just show indented empty space next to the left side of the shutdown pop-up menu options.  Seems like a waste of space that could be used to better visual effect.

**Comment by bonzibudd** [comment]
Yeah, not possible right now, at least not in this way. However, you can add standard icons to the shutdown items of the classic styles under `Customize Start Menu`.

Personally, I think a better feature would be the ability to completely customize the shutdown items in the 7 style, to be more similar to the classic styles. Since the shutdown box isn't technically a main2 item, it is unavailable in the customization section.

**Comment by kamyk2000** [comment]
I'd be better than happy to have the shutdown options from classic "customize start menu" being available in win7 style skins.  I wouldn't know from experience, but I'd imagine it would be easier to implement adding the pre-existing classic option to the pre-existing Win7 style section than to rewrite for a new bitmap.  I'd like to change my request to that instead, under the assumption it would be easier to code and would add visual appeal to the shutdown menu, which is what I really wanted anyway.

**Comment by bonzibudd** [comment]
Yep, I would have to agree. That way it would probably be easier for developers to add, and not a pain for skinners.
